accepted/PSR-16-simple-cache.md

```diff
@@ -178,7 +178,8 @@ interface CacheInterface
      * @return bool True if the item was successfully removed. False if there was an error.
      *
      * @throws \Psr\SimpleCache\InvalidArgumentException
-     *   MUST be thrown if the $key string is not a legal value.
+     *   MUST be thrown if the $key string is not a legal value 
+     *   OR if $ttl is neither an integer or a \DateInterval object.
```

Comment:
You are right, I pasted my comment above the wrong method. I've fixed this (changed from delete to the set method)